assets/queries/dockerCompose/ipc_set_to_host/metadata.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "baa3890f-bed7-46f5-ab8f-1da8fc91c729",
+  "queryName": "IPC Set To Host",
```

```suggestion
  "queryName": "Shared Host IPC Namespace"
```

Please, also update the query folder name